src/main/groovy/de/dkfz/eilslabs/batcheuphoria/IntegrationTestStarter.groovy

```diff
@@ -21,71 +22,95 @@ import de.dkfz.roddy.tools.BufferValue
 import de.dkfz.roddy.tools.LoggerWrapper
 import de.dkfz.roddy.tools.RoddyIOHelperMethods
 import de.dkfz.roddy.tools.TimeUnit
-import groovy.transform.CompileStatic
 
 /**
  * Starter class for BE integration tests.
  *
  * Created by heinold on 27.03.17.
  */
-@CompileStatic
+//@CompileStatic
 class IntegrationTestStarter {
 
     static LoggerWrapper log = LoggerWrapper.getLogger(IntegrationTestStarter)
     static TestExecutionService executionService
+    static RestExecutionService restExecutionService
+    static String testScript = "\"#!/bin/bash\\nsleep 15\\n\""
     static File batchEuphoriaTestScript
 
     static void main(String[] args) {
-        checkStartup(args)
-
-        initializeTests(args)
-
-        getOptionsFromArgs(args).each { runTestsFor(it) }
+        IntegrationTestInput testInput= new IntegrationTestInput()
+        checkStartup(args,testInput)
+
+        if(testInput.clusterSystem == AvailableClusterSystems.lsf){
+            initializeTests(testInput.account,testInput.server)
+            initializeLSFTest(testInput.restServer, testInput.restAccount)
+            runTestsFor(testInput.clusterSystem, restExecutionService)
+        }else{
+            initializeTests(testInput.account,testInput.server)
+            runTestsFor(testInput.clusterSystem, executionService)
+        }
 
         finalizeTests()
     }
 
-    private static List<AvailableClusterSystems> getOptionsFromArgs(String[] args) {
-        args[2..-1].collect { String it -> it.toLowerCase() as AvailableClusterSystems }
-    }
 
-    private static void runTestsFor(AvailableClusterSystems option) {
+    private static void runTestsFor(AvailableClusterSystems option, ExecutionService es) {
         JobManager jobManager
-        int maxSleep = 5
         log.always("Creating job manager instance for ${option}")
 
         try {
             jobManager = option.loadClass().getDeclaredConstructor(ExecutionService, JobManagerCreationParameters)
-                    .newInstance(executionService,
+                    .newInstance(es,
                     new JobManagerCreationParametersBuilder()
                             .setCreateDaemon(false)
                             .setTrackUserJobsOnly(true)
                             .build()
             ) as JobManager
         } catch (Exception ex) {
-            log.severe("Could not load and instantiate job manager class ${option.className}.")
+            log.severe("Could not load and instantiate job manager class ${option.className}. Exception: ${ex.printStackTrace()}")
```

Comment:
You don't need that. .severe can take exceptions. Please update the ToolLib if the message is not handled good enough there.